public/locales/en-US/send.ftl

```diff
@@ -50,6 +51,7 @@ downloadButtonLabel = Download
     .title = Download
 downloadNotification = Your download has completed.
 downloadFinish = Download Complete
+fileSizeProgress = ({ $partialSize } of { $totalSize })
```

Comment:
In general it's not a good idea to preland strings, because a) you won't have any way to test it and b) you might have to change your mind in the implementation phase.

Note: if this PR makes a string obsolete, that string should be removed.